CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Contributing to AI Validation
```

Comment:
What do you tihnk of making this a page on the MkDocs site itself? And set a reference to this file in the parent directory.

Reply:
Good question. i do not really care as long as github recognizes it

Reply:
Either direction should work, we can use softlinks, right?

Reply:
@anneschuth do you think we should publish it on our site? Or only keep it as contributing guidelines in the repo?

Reply:
I think we should do both. I'd probably symlink it from `/docs/Way-of-Working/Contributing.md` to `/Contributing.md`

Reply:
@berrydenhartog if you agree, can you make this change?